src/lib.rs

```diff
@@ -1,42 +1,47 @@
 #![deny(clippy::all)]
 
-use std::{ffi::c_ulonglong, os::raw::c_uchar};
+use hashtree_rs::{hash as HASH, init};
```

Comment:
It looks like this is published https://crates.io/crates/hashtree-rs

We can simplify by removing the git submodule, custom build.rs step.

Reply:
The code to fix the Mac and Linux arm builds is not on main yet. Once they are merged we can swap though